_opensearch/point-in-time-api.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: default
+title: Point in Time API
+nav_order: 58
+has_children: false
+parent: Point in Time
+---
+
+# Point in Time API
+
+Use the [Point in Time (PIT)]({{site.url}}{{site.baseurl}}/opensearch/point-in-time/) API to manage Points in Time. 
+
+---
+
+#### Table of contents
+- TOC
+{:toc}
+
+---
+
+## Create a PIT
+Introduced 2.4
+{: .label .label-purple }
+
+Creates a PIT. The `keep_alive` query parameter specifies how long to keep this PIT and is required.
```

Comment:
Creates a PIT. The required query parameter `keep_alive` specifies how long to keep this PIT.